packages/bpk-component-pagination/src/BpkPaginationPage.module.scss

```diff
@@ -18,43 +18,26 @@
 
 @use '../../unstable__bpk-mixins/tokens';
 @use '../../unstable__bpk-mixins/borders';
-@use '../../unstable__bpk-mixins/typography';
+@use '../../unstable__bpk-mixins/buttons';
```

Comment:
Using the button mixins allows us to keep these pagination buttons aligned with the BpkButtons.